crates/ruff_dev/src/generate_rules_table.rs

```diff
@@ -22,14 +22,16 @@ fn generate_table(table_out: &mut String, rules: impl IntoIterator<Item = Rule>,
     for rule in rules {
         let fix_token = match rule.fixable() {
             FixAvailability::Always | FixAvailability::Sometimes => {
-                format!("<span style='opacity: 1'>{FIX_SYMBOL}</span>")
+                format!("<span title='Automatically fixable'>{FIX_SYMBOL}</span>")
```

Comment:
I'd prefer to use "Fix available" to match our other language
```suggestion
                format!("<span title='Fix available'>{FIX_SYMBOL}</span>")
```